### ISSUE TITLE
UserAccounts needs a logo!

### ISSUE THREAD

**Issue by splendido** [meteor-useraccounts/core#174]
...at least for the organization here on GitHub! ;-)

**Comment by juanmnl** [comment]
I'd love to help with this. Any ideas or should i go freestyle-crazy-and-stuff?

**Comment by splendido** [comment]
Hey @juanmnl!

...I'm not a designer, so lets try whatever might come to your mind! ;-)
Thank you for volounteering on this!

**Comment by juanmnl** [comment]
@splendido Great! i have plenty of ideas, but first, the name you want is 'Meteor - User accounts' or just 'User accounts' ?

**Comment by splendido** [comment]
UserAccounts should be enough ;-)

**Comment by splendido** [comment]
Some silly proposal ;-)

![useraccounts_ico](https://cloud.githubusercontent.com/assets/6148980/6265860/05868892-b833-11e4-913d-78dcf29ea022.png)

**Comment by juanmnl** [comment]
Hey @splendido , don't think i've forgotten about this. I've just been extremely busy.
I made this sketch last week and i'm currently vectorising it. Hope i can finish it next week :) Tell me what you think.
![2015-02-19 18 35 25](https://cloud.githubusercontent.com/assets/699945/6278683/dd8ca9b4-b866-11e4-8bfd-4bdb453772e6.png)

**Comment by splendido** [comment]
Wow, looks terrific! :-)

**Comment by splendido** [comment]
...what if the meteor appearing on the bottom left resembled the official Meteor logo?

**Comment by juanmnl** [comment]
@splendido Of course, it fits well, but i didn't find the meteor logo in eps or something usable so i made mine [not exactly the same, was for testing purposes]
I changed the POV of the astronaut as i progressed. Like it better now. I'll add some detail to the body and a plate on its chest that would read USER Accounts. Hope you like it :smiley: 

![screen shot 2015-02-26 at 8 11 18 pm](https://cloud.githubusercontent.com/assets/699945/6405658/459279d4-bdf4-11e4-8e41-b1c5207a2e88.png)

**Comment by splendido** [comment]
Wow, really neat!

[Here](https://github.com/splendido/MIM2014Slides/blob/master/public/icons/meteor_icon.svg) is the Meteor logo in svg ;-)

**Comment by splendido** [comment]
...I have also the original black version, but I don't think it makes any difference
In cse, lets write me an email ;-)

**Comment by juanmnl** [comment]
Hey @splendido ! I'm sorry but i had a shoulder accident and haven't been able to pick up my computer for some weeks. Starting to feel better so i hope i can finish it up over the weekend and send it.
Cheers!

**Comment by splendido** [comment]
Oh, damn! I'm so sorry for you.

Take your time and don't care about useraccounts' logo, we have time!

My best wishes,
Luca

**Comment by splendido** [comment]
Hey @juanmnl, how are you doing with your shoulder?!?
I hope everything's getting back to normal!

**Comment by splendido** [comment]
I'm getting worried for your health... You ok?

**Comment by juanmnl** [comment]
Hey @splendido ! Getting much better now, thanks for asking. It's been a long month and a half. (we don't realise how much we use a muscle, until there's pain in every movement)
I'm finishing my recovery treatment next week and i'll be (finally) able to sit by the computer and code and draw and work in general. :smile:
I'll get back to this and finish it by the end of next week.

Cheers from Ecuador!

**Comment by splendido** [comment]
Oh, good news then!
I'm happy you're finishing your recovery :)

Cheers from Italy!

**Comment by juanmnl** [comment]
Hope you don't mind, i made a circular icon instead of a rectangle. I think it fits better. Also added some vintage camera lenses effects, as it gives different depths to the icon/logo.
I'll like to add even more detail with time, but i think it's usable for the project.
I have the vector logo (without any effect) made in SketchApp, and different exports from photoshop.

Tell me which sizes and formats you want the file and where to send them.

Cheers! :)

![ua_512x512-vintage](https://cloud.githubusercontent.com/assets/699945/7691100/702faefa-fd7d-11e4-8832-b243078d2952.png)
![ua_512x512-vintage-color-cast](https://cloud.githubusercontent.com/assets/699945/7691204/46a317c4-fd7e-11e4-94ac-c89c84c55f55.png)

**Comment by splendido** [comment]
Wow, it looks awesome! :-)
...and, welcome back!

well, I'm not a graphic designer so I have no idea which sizes do we need...
for now I'd like to set up the logo for the GitHub organization and perhaps put it on useraccounts.meteor.com

Then we'll have to post something about it on crater.io and forums.meteor.com!
...would you like to be you to write it?

Thanks a bunch!
...you'll deserve some rewards whenever this will be possible!!!

**Comment by splendido** [comment]
...I'm sorry, I could'n wait: https://github.com/meteor-useraccounts :-)

**Comment by juanmnl** [comment]
Hey! Really glad you liked it :smile:
Gonna make a dropbox folder with different sizes and original backups, and send you the link.
Not sure for the writing, but whatever you need give me a shout on twitter @ _juanmnl
No reward needed, always wanted to give something back to OSS, at least with a design.
Cheers!

**Comment by juanmnl** [comment]
@splendido Sent you an email with the dropbox folder link. (juanmnl@me.com)

**Comment by splendido** [comment]
got it!
awesome :-)

**Comment by venetianthief** [comment]
@juanmnl Nice job! I'm digging this a lot!

**Comment by steve-ross** [comment]
+1 for the first one: https://github.com/meteor-useraccounts/core/issues/174#issuecomment-103225281
